### ISSUE TITLE
crash when using a selector on drawableResId

### ISSUE THREAD

**Issue by yobijoss** [minibugdev/DrawableBadge#6]
If you set a selector as a drawableResId on DrawableBagde.Builder. Then app crashes and shows this error.

![image](https://user-images.githubusercontent.com/5233271/31644652-13ab56be-b2bd-11e7-925f-ec90a82e6317.png)

**Comment by yobijoss** [comment]
here is the code 

```
 DrawableBadge drawable = new DrawableBadge.Builder(getContext())
                .drawableResId(R.drawable.selector_notif_bell)
                .badgeColor(R.color.colorAccent)
                .badgeSize(R.dimen.badge_size)
                .badgePosition(BadgePosition.TOP_RIGHT)
                .textColor(R.color.white)
                .showBorder(true)
                .badgeBorderColor(R.color.white)
                .badgeBorderSize(R.dimen.badge_border_size)
                .build();

```

**Comment by minibugdev** [comment]
Thanks for report.
On version `1.0.1` we doesn't support to set drawable with `drawable selector`.
I'll add this feature to next release or you can create PR to fix the issue :)

**Comment by yobijoss** [comment]
This library is pretty cool and recent, If I make it to the fix I'll fire a PR hehe 
Thanks

**Comment by minibugdev** [comment]
Fixed on version 1.0.2 (https://github.com/minibugdev/DrawableBadge/pull/10)
